modules/data_cleaner_identification_difficulty/plugins/data_cleaner_identification_difficulty.php

```diff
@@ -62,14 +62,6 @@ function data_cleaner_identification_difficulty_data_cleaner_rules() {
   and vrdini.header_name='INI'
   and vrdini.key=vr.id_difficulty
   and vrdini.key::int>1 and vrdini.deleted=false
--- join to exclude warning user's who've already ID'd this species successfully.
```

Comment:
In removing the join to table alias o2, should you also remove the following from the return array:
```php
'where' => 'o2.id is null',
```

Reply:
Blimey - well spotted. I've updated the branch.